gnosis/safe-cmm-app-react
# Feature/#178/on new block event handler

# Description

Update balances/strategies on every new block

# To Test
1. Do the usual flow: deploy, withdraw, claim

- [ ] Should work as expected

2. On deploy page, check the console

- [ ] At every new block (few seconds) should log one msg regarding balances been updated and one regarding strategies

3. Change to strategies page

- [ ] At every new block (few seconds) should log one msg regarding strategies been updated

4. Change back to deploy page

- [ ] At every new block (few seconds) should log one msg regarding balances been updated and one regarding strategies

# Background

N/A
